app/deliver_grant_funding/routes/reports.py

```diff
@@ -325,9 +326,41 @@ def list_submissions(grant_id: UUID, report_id: UUID, submission_mode: Submissio
         grant=report.grant,
         report=report,
         helper=helper,
+        submission_mode=submission_mode,
     )
 
 
+@deliver_grant_funding_blueprint.route(
+    "/grant/<uuid:grant_id>/report/<uuid:report_id>/submissions/<submission_mode:submission_mode>/export/<export_format>",
```

Comment:
Plenty in the commit messages about this not yet being finalised or optimised but just a thought for the future when we know more about the need and have the full requirements, this `export_format` feels a bit like its trying to recreate Accept/ Content-Type headers which hopefully it wouldn't be too much effort to just hook in here, the client could then specify if it wants `text/csv`, `application/json`, `application/vnd.ms-excel` and these methods could route based on which was provided